pkgs/servers/zeronsd/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, rustfmt
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "zeronsd";
+  version = "0.5.2";
+
+  src = fetchFromGitHub {
+    owner = "zerotier";
+    repo = "zeronsd";
+    rev = "v${version}";
+    hash = "sha256-TL0bgzQgge6j1SpZCdxv/s4pBMSg4/3U5QisjkVE6BE=";
+  };
+
+  cargoHash = "sha256-WGap0j90obpJHiMNokCWg0Q3xIAqwvmiESg9NVnFMKE=";
+
+  nativeBuildInputs = [
+    pkg-config
+    rustfmt
+  ];
+
+  RUSTFMT = "${rustfmt}/bin/rustfmt";
+
+  doCheck = false;
```

Comment:
can you add a comment saying why you disabled the tests?

Reply:
Done. The Integration tests create test networks via the central API, and thus need a API token which we don't have at this point.